tests/api2/stats.py

```diff
@@ -12,9 +12,9 @@
 source_list = []
 source_dict = {}
 sources = GET('/stats/get_sources/').json()
-
 for source, types in sources.items():
-    source_list.append(source)
+    if 'interface-vnet0' not in source:
+        source_list.append(source)
```

Comment:
Whats the point in not adding vnet0?

Reply:
that is the interface-vnet0 tests that failed in Jenkins.

Reply:
Got it, its because of jails.